react/features/base/dialog/components/DialogContainer.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Implements a DialogContainer that will be responsible for
+ * showing all dialogs. We will need a separate container so we can handle
+ * multiple dialogs, showing them simultaneously or queueing them.
+ */
+export class DialogContainer extends Component {
+
+    /**
+     * DialogContainer component's property types.
+     *
+     * @static
+     */
+    static propTypes = {
+        /**
+         * The component to render.
+         */
+        _component: React.PropTypes.func,
```

Comment:
Is there some rule about making props private? I see you and @hristoterezov are on a different opinion for show and _show.